docs/deployment.md

```diff
@@ -1,8 +1,12 @@
 # Deployment
 
-### A foreword on security
+!!! warning "A foreword on security"
```

Comment:
👍

Reply:
These are awesome. [Here](https://squidfunk.github.io/mkdocs-material/extensions/admonition/) is the documentation on how to use them and the different styles.

Reply:
Please can you raise a small PR and drop a link to this in the README.md file?

Reply:
Will do